build/README.md

```diff
@@ -0,0 +1,39 @@
+## Runing inside docker
+
+```bash
+docker run \
+  -it --name kubefirst  \
+  --dns="1.0.0.1" --dns="208.67.222.222" --dns="8.8.8.8" \
+  -v $(PWD):/opt/kubefirst \
+  -v $HOME/.aws:/home/developer/.aws \
+   pagottoo/kubefirst_cli:1.8.5
```

Comment:
should we make this
```suggestion
   kubefirst/kubefirst:1.8.6
```
and i'll publish 1.8.6 to dockerhub as part of the release?

Reply:
yep, thanks.